modules/openapi-generator-gradle-plugin/src/test/kotlin/GenerateTaskDslTest.kt

```diff
@@ -29,6 +29,66 @@ class GenerateTaskDslTest : TestBase() {
         }
     """.trimIndent()
 
+    @Test
+    fun `openApiGenerate should create an expected file structure from URL config`() {
+        val specUrl = "https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master" +
```

Comment:
It might be a good idea to ping this to a specific commit or tag. That way changes to the remote spec won't affect the outcome of the test.